packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -32,20 +36,26 @@
             <div
                 v-for="(subStatusLine, index) in card.description"
                 :key="index"
+                v-make-text-accessible
                 :class="[$style['c-stampCard1-subStatusText']]"
                 :data-test-id="testIdForSection('subStatusText', index)">
                 {{ subStatusLine }}
             </div>
             <div
                 :class="[$style['c-stampCard1-expiryInfo']]"
                 :data-test-id="testIdForSection('expiryInfo')">
-                {{ expiryLine }}
+                {{ card.expiryLine }}
+                <template v-if="hasValidExpiryDate">
+                    <span :aria-label="expiryDateAccessible"><span aria-hidden="true">{{ expiryDateVisual }}</span></span>
+                </template>
             </div>
         </div>
         <div
             v-else
             :class="[$style['c-stampCard1-stamps']]"
-            :data-test-id="testIdForSection('stamps')">
+            :data-test-id="testIdForSection('stamps')"
+            role="img"
+            :aria-label="stampCardsStatusCopy()">
```

Comment:
Should this be just `:aria-label="stampCardsStatusCopy"`?

Reply:
have made it a computed property aswell